### PR TITLE
[SCSS][PDN-664] Chips

### PR DESCRIPTION
**Pull request by RipripKipkip**
image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1058

File: packages/scss/src/components/_chips.scss [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
+		background: _color("primary", "600");
+	}
+	&:focus {
+		box-shadow: 0 0 1px 2px _color("primary", "400");
+		outline: none;
+	}
+	&.is-disabled {
+		background: _color("primary", "500");
+		color: _color("primary", "300");
+		pointer-events: none;
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/_chips.scss:20]: Avec ça tu empêches la fermeture via le bouton non ?

Comment (lucienbertin) [reply packages/scss/src/components/_chips.scss:20]: english

Comment (sanlucca) [reply packages/scss/src/components/_chips.scss:20]: Isn't it weird to have a disable state on a chip ? It feels more like a readOnly and it shouldn't blocks pointer events, but just remove the chip kill. Keep the pointer events to allow text selection here

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1058

**Pull request merged**